geode-wan/build.gradle

```diff
@@ -24,7 +24,7 @@ dependencies {
   compile(platform(project(':boms:geode-all-bom')))
   gradleLint.ignore {
     // See GEODE-6128 -- ignore xml-apis in linter to avoid changes with every run.
-    upgradeTestCompile('xml-apis:xml-apis:1.4.01')
+    upgradeTestCompile('xml-apis:xml-apis:2.0.2')
```

Comment:
Lets take this opportunity to move this version into our dependency manager, instead of being the sole hard-coded number.